Gemfile.lock

```diff
@@ -336,7 +336,7 @@ GEM
       execjs
     coffee-script-source (1.12.2)
     colorize (0.8.1)
-    commonmarker (0.23.6)
+    commonmarker (0.23.9)
```

Comment:
I see that in the original PR we have `commonmaker` in v0.23.8, and in this one is on v0.23.9. I think that as long as are in the same minor version it should not be a problem. 

We probably should bump the commonmaker version in develop so its newer than the one that we have in v0.27, just to be extra sure that we don't introduce an issue here. Maybe I'm just too paranoid, but it'd be great to have some general guidelines as maintainers.